### ISSUE TITLE
Replace Data Plan Procurement Self-Service Instructions PDF with updated version

### ISSUE THREAD

**Issue by mrose914** [cal-itp/mobility-marketplace#537]
We link to the Data Plan Self-Service Instructions PDF in two places on [/how-to/get-connected](https://www.camobilitymarketplace.org/how-to/get-connected)

I updated the PDF based on a minor issue in Accessibility Check (Missing Title). Please use attached version for the two places it's linked: 
- Under **How do I get started?**
- Under **Where can I access the data plan rates?**

Also please change copy on page from "If you are interested in purchasing the FirstNet data plans, self-serve instructions can be found [here](https://resources.calitp.org/mobility-marketplace/FirstNet-Self-Service-Instructions.pdf)." TO If you are interested in purchasing the FirstNet data plans, please review our [Self Service Instructions.](https://resources.calitp.org/mobility-marketplace/FirstNet-Self-Service-Instructions.pdf).

**Comment by thekaveman** [comment]
Note for @allejo: this PDF document is hosted within a GCP bucket mapped to the `resources.calitp.org` domain.

I just gave you access as a **Cloud Storage Admin** so you should be able to login to GCP and work with the files in that bucket, here: https://console.cloud.google.com/storage/browser/resources.calitp.org;tab=objects?forceOnBucketsSortingFiltering=true&project=resources-calitp-org&prefix=&forceOnObjectsSortingFiltering=false

**Comment by allejo** [comment]
Ugh, I slipped up and accidentally pushed directly to main; sorry about that! But I have uploaded the new PDF to the GCP bucket and made the change requested in 18f903638a6db9f2f7417253739707fd45483f9f.

If I can get your eyes on this change, @mrose914, then this ticket can be closed (either by you or me) in place of an explicit approval in a PR.
